attribution-graphs-frontend/attribution_graph/init-cg-subgraph.js

```
@@ -0,0 +1,634 @@
+          .on('blur', save)
+          .on('keydown', ev => {
+            if (ev.key === 'Enter'){
+              save()
+              input.node().blur()
+            }
+            ev.stopPropagation()
+          })
+
+        input.node().focus()
+
+        function save(){
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)